spec/features/workflow/publish_without_selection_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.feature "Attempting to publish without confirming a review status" do
+  scenario do
+    given_there_is_an_edition_in_draft
+    when_i_visit_the_summary_page
+    and_try_and_publish_without_selecting_a_review_status
+    then_an_error_is_displayed
+  end
+
+  def given_there_is_an_edition_in_draft
+    @edition = create(:edition, :publishable)
+  end
+
+  def when_i_visit_the_summary_page
+    visit document_path(@edition.document)
+  end
+
+  def and_try_and_publish_without_selecting_a_review_status
+    click_on "Publish"
+    stub_publishing_api_publish(@edition.content_id, update_type: nil, locale: @edition.locale)
```

Comment:
I don't think we need to have this stub here, as hopefully we won't be doing anything :-).

Reply:
Yep you're correct!